### ISSUE TITLE
Neo4jSingleton purposely deletes a DB before instantiating it.

### ISSUE THREAD

**Issue by gfokkema** [DNAinator/dnainator#43]
This is done intentionally to keep our initial demo simple.
This has to be fixed in order to get persistence after closing the application.

**Comment by gfokkema** [comment]
Some improvements have been made in #47 regarding this issue.
The UI as well as the Neo4j graph should be updated to handle these changes.

The UI only has to include functionality to:
- allow the user to select whether he wants to delete the db
- allow the user to select a path to his db (and pass this value to the LoadService)

The Neo4j graph should be able to:
- detect whether indices are already created

**Comment by Unia** [comment]
Is still still relevant after #116 ?

**Comment by gfokkema** [comment]
I think this issue should be superseded by #118, therefore I'm closing this.
